Library/Styles/BaseStyles.swift

```diff
@@ -37,7 +37,7 @@ public let baseNavigationBarStyle =
     NSFontAttributeName: UIFont.ksr_callout()
   ]
   <> UINavigationBar.lens.translucent .~ false
-  <> UINavigationBar.lens.barTintColor .~ .ksr_grey_100
+  <> UINavigationBar.lens.barTintColor .~ .white
```

Comment:
note, these styles are updated in the Global Styles PR I have up, which I'd like to close before this one.